lib/compress.js

```diff
@@ -1838,6 +1838,11 @@ merge(Compressor.prototype, {
     });
 
     OPT(AST_If, function(self, compressor){
+        if (self.alternative instanceof AST_EmptyStatement) {
+            self.alternative = null;
+            // fall through
+        }
+
```

Comment:
Would this overlap in functionality with [the line below](https://github.com/mishoo/UglifyJS2/pull/1449/files#diff-230c53e13e3ee367aec7d49470eb42caL1869)?

Also, `is_empty()` seems to check a few more cases than just `AST_EmptyStatement`. May be just move that line up here?